### ISSUE TITLE
How to use this app as a launchpad?

### ISSUE THREAD

**Issue by souradeepDe225** [anzbert/beat_pads#43]
Thanks for this great app! ❤️

I wish to use it as a launchpad for ableton live, can you please share some steps?

**Comment by anzbert** [comment]
Hi

So the original Launchpad works as a controller for ableton because they provide whats called a control script for Ableton. I guess one could be made for this app as well and I may look into it, as per these [instructions](https://help.ableton.com/hc/en-us/articles/206240184-Creating-your-own-Control-Surface-script).
You can still use the app to launch clips right now, if you midi map the pads to clips in Ableton's session view. That way is just a little less flexible, I guess. Anyway, I may look into the control script thing at some point. But feel free to experiment with that as well.

**Comment by souradeepDe225** [comment]
Yes, just to get that beat synced glow like launchpad, I had to create a dummy MIDI clip which sends(MIDI to) the exact same notes (in counts) to the app as it was used to trigger the scene/clip.

And also by any chance do you know if I can send the labels on the scene/clip to come to the pad labels on app? Is it possible by MIDI or is it possible only by those remote scripts?

**Comment by anzbert** [comment]
I would say that's only possible with remote scripts, and even with those
it would need to be sent with a custom midi sysex message, I believe,
without really having looked into it.

**Comment by anzbert** [comment]
Maybe I could implement receiving midi clock and flashing on each 4th note
?!? Would that be desirable?

**Comment by souradeepDe225** [comment]
Yup, this sounds cool.

**Comment by souradeepDe225** [comment]
also @anzbert , if you could add an option to enter custom labels to each pad, that would be great? 😊

**Comment by souradeepDe225** [comment]
> also @anzbert , if you could add an option to enter custom labels/colors to each pad, that would be great? 😊

Hi @anzbert , did you get a chance to look into this?

**Comment by anzbert** [comment]
No not yet.. sorry.

Only been doing minor fixes recently.
